### DISCUSSION TITLE
I want to add a new Funtion can anyone tell me how to add !?

### DISCUSSION THREAD

**Discussion by makstyle119** [js-fns/date-fns#3541] [Show and tell] [answered]
I want to add a new Function can anyone tell me how to add it!?

**Comment by tan75** [comment, marked as answer]
Here is the [Contribution guide](https://date-fns.org/v2.30.0/docs/Contributing).
If you want to add a new function to this package, please submit a PR.

**Reply by makstyle119** [reply]
I have created a PR: https://github.com/date-fns/date-fns/pull/3540 
I will update this PR as per contribution guidance.
Thanks for the help
